tests/test_compare_rules.py

```
@@ -0,0 +1,110 @@
+from pathlib import Path
+
+import pytest
+
+from evals.scripts.compare_rules import (
+    deterministic_fails,
+    load_cases,
+    _REFUSAL_LENGTH_THRESHOLD,
+)
+
+
+def _case(**checks):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_case`

(ANN202)